<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IqtTab</class>
 <widget class="QWidget" name="IqtTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="gbInput">
     <property name="title">
      <string>Input</string>
     </property>
     <layout class="QGridLayout" name="layoutFiles">
      <item row="2" column="0">
       <widget class="QLabel" name="lbResolution">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Resolution</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbSample">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Sample</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="MantidQt::MantidWidgets::DataSelector" name="dsInput" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="loadLabelText" stdset="0">
         <string>Plot Input</string>
        </property>
        <property name="workspaceSuffixes" stdset="0">
         <stringlist>
          <string>_red</string>
          <string>_sqw</string>
         </stringlist>
        </property>
        <property name="fileBrowserSuffixes" stdset="0">
         <stringlist>
          <string>_red.nxs</string>
          <string>_sqw.nxs</string>
         </stringlist>
        </property>
        <property name="showLoad" stdset="0">
         <bool>false</bool>
        </property>
        <property name="ShowGroups" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="MantidQt::MantidWidgets::DataSelector" name="dsResolution" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="autoLoad" stdset="0">
         <bool>true</bool>
        </property>
        <property name="workspaceSuffixes" stdset="0">
         <stringlist>
          <string>_res</string>
          <string>_red</string>
          <string>_sqw</string>
         </stringlist>
        </property>
        <property name="fileBrowserSuffixes" stdset="0">
         <stringlist>
          <string>_res.nxs</string>
          <string>_red.nxs</string>
          <string>_sqw.nxs</string>
         </stringlist>
        </property>
        <property name="showLoad" stdset="0">
         <bool>false</bool>
        </property>
        <property name="ShowGroups" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="loFury" stretch="1,1">
     <item>
      <layout class="QVBoxLayout" name="properties"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,5,3">
       <item>
        <widget class="QFrame" name="frame_2">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="leftMargin">
           <number>30</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="ckSymmetricEnergy">
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Untick to allow an asymmetric energy range.</string>
            </property>
            <property name="text">
             <string>Symmetric Energy Range</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="lbSpectrum">
            <property name="text">
             <string>Spectrum:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spPreviewSpec"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="MantidQt::MantidWidgets::PreviewPlot" name="ppPlot" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>125</height>
          </size>
         </property>
         <property name="canvasColour" stdset="0">
          <color>
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </property>
         <property name="showLegend" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbPlotPreview">
       <property name="text">
        <string>Plot Current Preview</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="enEnforceNormalization">
     <property name="toolTip">
      <string>Normalization to enforce I(t=0)</string>
     </property>
     <property name="text">
      <string>Enforce Normalization</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ErrorCalc">
     <property name="title">
      <string>Monte Carlo Error Calculation</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>7</number>
      </property>
      <item>
       <widget class="QCheckBox" name="cbCalculateErrors">
        <property name="toolTip">
         <string>Untick to skip the calculation of errors.</string>
        </property>
        <property name="text">
         <string>Calculate Errors</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Number of Iterations:</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft">
       <widget class="QSpinBox" name="spIterations">
        <property name="toolTip">
         <string>The number of iterations to use when calculating errors.</string>
        </property>
        <property name="minimum">
         <number>3</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="MantidQt::CustomInterfaces::RunView" name="runWidget" native="true"/>
   </item>
   <item>
    <widget class="QGroupBox" name="gbOutput">
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>7</number>
      </property>
      <item>
       <widget class="MantidQt::CustomInterfaces::OutputPlotOptionsView" name="ipoPlotOptions" native="true"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbSave">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Save Result</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::CustomInterfaces::RunView</class>
   <extends>QWidget</extends>
   <header>MantidQtWidgets/Spectroscopy/RunWidget/RunView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MantidQt::CustomInterfaces::OutputPlotOptionsView</class>
   <extends>QWidget</extends>
   <header>MantidQtWidgets/Spectroscopy/OutputWidget/OutputPlotOptionsView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MantidQt::MantidWidgets::PreviewPlot</class>
   <extends>QWidget</extends>
   <header>MantidQtWidgets/Plotting/PreviewPlot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MantidQt::MantidWidgets::DataSelector</class>
   <extends>QWidget</extends>
   <header>MantidQtWidgets/Common/DataSelector.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
